tests/phpunit/feature/TestBooleanSearchOperators.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * Test Boolean Search Operators feature
+ *
+ * @since  1.2.0
+ * @package ElasticPressLabs
+ */
+
+namespace ElasticPressLabsTest;
+
+use ElasticPressLabs;
+
+/**
+ * Boolean Search Operators test class
+ *
+ * @since  1.2.0
+ */
+class TestBooleanSearchOperators extends \WP_UnitTestCase {
+	/**
+	 * Setup each test.
+	 *
+	 * @since  1.2.0
+	 */
+	public function setUp() {
+		$instance = new ElasticPressLabs\Feature\BooleanSearchOperators();
+		\ElasticPress\Features::factory()->register_feature($instance);
+	}
+
+	/**
+	 * Get Boolean Search Operators feature
+	 *
+	 * @since  1.2.0
+	 * @return BooleanSearchOperators
+	 */
+	protected function get_feature() {
+		return \ElasticPress\Features::factory()->get_registered_feature( 'boolean_search_operators' );
+	}
+
+	/**
+	 * Test constrcut
+	 *
+	 * @since  1.2.0
+	 */
+	public function testConstruct() {
+		$instance = $this->get_feature();
+
+        $this->assertEquals( 'boolean_search_operators', $instance->slug );
+        $this->assertEquals( 'Boolean Search Operators', $instance->title );
```

Comment:
Indentation seems a little off here. Do you think we could fix it @Rahmon?

Reply:
I think it's something with Github because locally looks fine :man_shrugging: